test/conformance/ptp/ptp.go

```diff
@@ -31,10 +32,17 @@ import (
 	testclient "github.com/openshift/ptp-operator/test/utils/client"
 	"github.com/openshift/ptp-operator/test/utils/execute"
 	"github.com/openshift/ptp-operator/test/utils/nodes"
+	testnode "github.com/openshift/ptp-operator/test/utils/nodes"
 	"github.com/openshift/ptp-operator/test/utils/pods"
 	"github.com/openshift/ptp-operator/test/utils/testconfig"
 )
 
+const (
+	defaultNamespace        = "default"
```

Comment:
I am creating the reboot daemonset (unrelated to ptp config) in some already existing (in this case, default) namespace in the openshift cluster. And this DS is being deleted after the test. So, the namespace does not matter here, right?

Reply:
@edcdavid Could you respond here the reason for using only default namespace here?

Reply:
We are using the default namespace for the daemonset since the default namespace already has enough permissions to run the "oc debug" command. If not using the default namespace we'd have to configure privileged mode for this namespace when we otherwise would not need it ( e.g. oc adm policy add-scc-to-group privileged system:authenticated ).The daemonset is not related to ptp, it is just a way to access the platform

Reply:
ok this is not for PTP. Thanks

Reply:
Interestingly, in `cnfdt13` cluster, we have permission to run the daemonset with privilege permission in `openshift-ptp` namespace, but in 3 nodes cluster not. 
For other clusters (if new are coming up), can we ensure that only `default` namespace would be having necessary permissions at service account level for the daemonset?